control_msgs/action/AntipodalGripperCommand.action

```diff
@@ -0,0 +1,16 @@
+# Antipodal grippers refer to an end effector where two opposing fingers grasp an object from opposite sides.
+sensor_msgs/JointState command # The velocity and effort fields are optional and not used if empty.
+---
+sensor_msgs/JointState state # The current gripper state.
+# position refers to the gap size (in meters).
+# Optional: velocity refers to the Cartesian velocity of the finger right finger.
+# effort refers to the current effort exerted (in Newtons)
```

Comment:
```suggestion
# position refers to the position of each joint (radians or meters).
# Optional: velocity of each joint (radians or meters / Second).
# Optional: effort of each joint (Newtons or Newton-meters)
```

As we discussed I don't think it makes sense to command `distance` between the fingers because they are not a physical joint on the robot and the result can be interpenetrated in many different ways (especially if the user is able to manufacture custom fingers).

Reply:
Done